app/helpers/content_helper.rb

```diff
@@ -2,4 +2,8 @@ module ContentHelper
   def path_for_this_page(page)
     "#{content_url}/#{page.slug}"
   end
+
+  def has_active_css(href)
+    "app-mobile-nav__subnav-item--current" if current_page?(href)
+  end
```

Comment:
Yesterday I added a function to do this;

```
  def is_selected(page)
    page.slug == params[:slug]
  end

It is already in main, can you use this (its called is_selected) ? Rather than duplicate it ? Then put the css in the erb file, so that its close to the html. And in the html, use is_selected to conditionally use the css.

I just think its easier to follow if the css is in the erb, not the helper

Reply:
ok ive switched to using your function, and added erb conditional rendering for class but renamed the method is_current? - ? is ruby convention for boolean and the css classes we need for this use current in their name.